xbmc/video/dialogs/GUIDialogVideoVersion.cpp

```diff
@@ -90,6 +91,12 @@ bool CGUIDialogVideoVersion::OnMessage(CGUIMessage& message)
 {
   switch (message.GetMessage())
   {
+    case GUI_MSG_WINDOW_INIT:
+    {
+      m_cancelled = false;
```

Comment:
is this needed? m_cancelled should default to false I guess

Reply:
The dialog instance survives close and reopen of the dialog so the member needs to be re-inited. Do you have a mother piece in mind to do this?

Reply:
Ah right, disregard then